episodes/files/my_first_bash_script.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash
+# This script loops through .txt files, returns the
+# file name, first line, and last line of the file
 
-for filename in *.txt
+for file in *.txt
```

Comment:
In this change, however, I would suggest keeping the variable as filename. The problem, as you noted, is the missing quotes.